README.md

```diff
@@ -51,6 +51,11 @@ Then add **hubot-huntflow-reloaded** to your `external-scripts.json`:
 |`REDIS_PORT`                | Port Redis listens on.                                                                        | 16379                     |
 |`REDIS_PASSWORD`            | Specifies the Redis password.                                                                 | null                      |
 |`REDIS_CHANNEL`             | Defines the name of Redis channel to get messages from the server.                            | `hubot-huntflow-reloaded` |
+|`BASE_SERVER_URL`           | Defines the server url to handle requests.                                                    | `http://127.0.0.1:8888/`  |
+|`SERVER_USER_EMAIL`         | Defines the server user email to make authorized requests.                                    | null                      |
+|`SERVER_USER_PASSWORD`      | Defines the server user password to make authorized requests.                                 | null                      |
+
+**Note:**  Be sure that  you specified `SERVER_USER_EMAIL` and `SERVER_USER_PASSWORD` to handle huntflow control requests. 
```

Comment:
Let's include the warning in the 'Prerequisites' section above, where other required params are listed.